Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
@mikemcquaid I am sorry I used that term in relation to you, Mike. And I am sorry to hear you have a stressful life at the moment, I really am, and I appreciate your unwavering dedication to the project while under stress.

I thought you were satisfied with the PR'd code [previously](https://github.com/Homebrew/homebrew-core/pull/464#discussion_r60416826), so a code change request later felt offhand and frustrating.

Either way, thank you for going out of your way to merge my submission.